### ISSUE TITLE
Notion: Folders starting with a dot fails

### ISSUE THREAD

**Issue by Sire** [obsidianmd/obsidian-importer#87]
`
caught (in promise) Error: Failed to create folder at "Notion/Dev/.NET  Web/"
    at Bn.createFolders (plugin:obsidian-importer:18:2440)
    at async Bn.import (plugin:obsidian-importer:72:4554)
    at async HTMLButtonElement.eval (plugin:obsidian-importer:72:10602)
createFolders @ plugin:obsidian-importer:18
`

**Comment by joshuatazrein** [comment]
Fixed with:
```
async createFolders(path: string): Promise<TFolder> {
		// can't create folders starting with a dot
		const sanitizedPath = path.replace(/^\./g, '').replace(/\/./g, '/')
```

**Comment by nonodesu** [comment]
@joshuatazrein will that fix also handle folders that start with several dots in a row? For example I have some pages named '... continued'.

**Comment by joshuatazrein** [comment]
It's been updated to include several dots

**Comment by ericaxu** [comment]
@joshuatazrein Can I close this issue now? Thanks!

**Comment by joshuatazrein** [comment]
Yep!

**Comment by kepano** [comment]
Closed via [1.5.5](https://github.com/obsidianmd/obsidian-importer/releases/tag/1.5.5)
